pommerman/agents/player_agent_blocking.py

```diff
@@ -0,0 +1,44 @@
+"""
+This variant is blocking, that is the game pauses for keyboard input.
+"""
+
+from time import time
+import click
+
+from . import BaseAgent
+from .. import characters
+from .. import constants
+
+# keypad control codes
+K_PREFIX='\x1b'
+K_RT='[C'
+K_LF='[D'
+K_UP='[A'
+K_DN='[B'
+
+class PlayerAgentBlocking(BaseAgent):
+    """Block for keyboard input."""
+
+    def __init__(self, character=characters.Bomber, agent_control='arrows'):
+        super(PlayerAgentBlocking, self).__init__(character)
+        self.agent_control=agent_control
+
+    def act(self, obs, action_space):
+        key = click.getchar()
+        if self.agent_control=='arrows':
+            if key==K_PREFIX+K_RT: return constants.Action.Right.value
+            if key==K_PREFIX+K_LF: return constants.Action.Left.value
+            if key==K_PREFIX+K_UP: return constants.Action.Up.value
+            if key==K_PREFIX+K_DN: return constants.Action.Down.value
```

Comment:
nit: please use an enum or make it explicit. it's much nicer and more legible to have it read `k.UP`, `k.RIGHT`, ...